presto-main/src/test/java/io/prestosql/operator/scalar/AbstractTestRegexpFunctions.java

```diff
@@ -236,4 +237,79 @@ public void testRegexpSplit()
         assertFunction("REGEXP_SPLIT(',,a,,,b,c,d,,', ',')", new ArrayType(createVarcharType(13)), ImmutableList.of("", "", "a", "", "", "b", "c", "d", "", ""));
         assertFunction("REGEXP_SPLIT(',,,', ',')", new ArrayType(createVarcharType(3)), ImmutableList.of("", "", "", ""));
     }
+
+    @Test
+    public void testRegexpCount()
```

Comment:
Let’s add a helper `assertRegexpCount(input, pattern, expectedCount)`. It should also check that `cardinality(regexp_extract_all(...))` produces the same result as we expect these to be equivalent (especially for edge cases). Check if there are any test cases for the extract functions that we should have here.